docker/build_artifacts/sagemaker/serve.py

```diff
@@ -324,6 +327,24 @@ def _wait_for_gunicorn(self):
                 log.info("gunicorn server is ready!")
                 return
 
+    def _wait_for_tfs(self):
+        # Wait until tfs server is up and running
+        while True:
+            try:
+                tfs_ready_count = 0
+                for i in range(self._tfs_instance_count):
+                    tfs_url = "http://localhost:{}/v1/models/{}/metadata" \
+                        .format(self._tfs_rest_port[i], self._tfs_default_model_name)
```

Comment:
This looks suspicious - shouldn't there be a list/dict of corresponding model names (potentially different) for each TF server?

Reply:
All these TF servers are using the same model here. If it is multi-model endpoint, the tensorflow server is not started during container initialization.